cinch/roles/jenkins_master/templates/etc/nginx/conf.d/jenkins_https.conf

```diff
@@ -0,0 +1,65 @@
+server_names_hash_bucket_size 6400;
+upstream jenkins {
+  server 127.0.0.1:8080 fail_timeout=0;
+}
+
+server {
+  listen 80;
+  server_name {{ service_name }};
+  return 301 https://$host$request_uri;
+}
+
+server {
+  listen 443 ssl;
+  server_name {{ service_name }};
+  gzip on;
+    ssl_certificate {{ (jenkins_ssl_cert is defined) | ternary('/var/lib/jenkins/.ssl/jenkins.crt', '/etc/nginx/conf.d/ssl.pem') }};
```

Comment:
Now that keytool is out of the picture, is there any need to store certificates in $JENKINS_HOME/.ssl? I always have seen that path used in conjunction with keytool, and to me it seems that we can just always store certificates where nginx expects to read them by default.